tests/std/tests/Dev11_0000000_user_defined_literals/test.cpp

```diff
@@ -402,9 +402,9 @@ int main() {
         std::experimental::erase_if(str1, is_vowel{});
         assert(str1 == "ct flffy kttns");
 
-        std::string str2{"visual studio"};
-        std::experimental::erase(str2, 'u');
-        assert(str2 == "visal stdio");
+        std::string str2{"broad anticausotic"};
+        std::experimental::erase(str2, 'o');
+        assert(str2 == "brad anticaustic");

```

Comment:
I don't object to the intent of this change, but it's being performed inconsistently - the second example differs, and results in only one letter being removed. (Also, "anticausotic" is difficult to recognize as an actual word; apparently it's an archaic synonym for "antipyretic".) The original examples were identical and removed two letters. I suggest:

```suggestion
        std::string str2{"asynchronous beat"};
        std::experimental::erase(str2, 'a');
        assert(str2 == "synchronous bet");
```

Reply:
Thanks, I was having a hard time finding things that worked.